ephios/static/ephios/js/areyousure_form.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function () {
+    $('form').areYouSure();
+    // Listener for inputs that can dynamically add new forms and trigger a scan of the new form
+    $('.add-form').click(function() {
+        $('form').trigger('reinitialize.areYouSure');
+      });
+});
```

Comment:
Couldn't we just look for forms (or their submit buttons) in `main.js` and apply areyousure there? I saw this here:

https://github.com/pretix/pretix/blob/a897f60fc5a22d9eb9bf4107111eb8ea05d09b3b/src/pretix/static/pretixcontrol/js/ui/main.js#L874

That way, we wouldn't need to (remember to) include areyousure in every form based view. Also, if we get false positives, we could add a class to exclude those particular elements. What do you think?

Reply:
Sure thing! I'll update and push those changes.